integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseCatalog.java

```diff
@@ -106,13 +108,15 @@ public void initialize(String name, Map<String, String> properties) {
     String truststore = properties.getOrDefault(TRUST_STORE, "");
     String token = properties.getOrDefault(AUTH_TOKEN, null);
     String httpConnectionStrategy = properties.getOrDefault(HTTP_CONNECTION_STRATEGY, null);
+    String clientName = properties.getOrDefault(CLIENT_NAME, null);
```

Comment:
if properties map includes CLIENT_NAME, e.g. here: https://github.com/linkedin-multiproduct/trino-plugins/blob/a48ca1ac698426ce216eb93a03ee1f79bc7a945a/trino-metacat/src/main/java/com/linkedin/trino/metacat/OpenHouseCatalogClient.java#L220

then we will pass them to the tableApiClient